Gems/ROS2/Code/Source/RobotImporter/Utils/TypeConversions.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "AzCore/Math/Transform.h"
+#include "RobotImporter/URDF/UrdfParser.h"
+#include <AzCore/Math/Color.h>
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/Math/Vector3.h>
+
+namespace ROS2::URDF
+{
+    //! Common types conversion between urdf and AZ formats
+    class TypeConversions
```

Comment:
As mentioned in other comments, this should be a namespace, not a class, if it only has static methods and its purpose is to group similar functions